packages/features/ee/workflows/lib/reminders/templates/customTemplate.ts

```diff
@@ -1,54 +1,84 @@
 import { guessEventLocationType } from "@calcom/app-store/locations";
 import type { Dayjs } from "@calcom/dayjs";
-import type { Prisma } from "@calcom/prisma/client";
+import { APP_NAME, WEBAPP_URL } from "@calcom/lib/constants";
+import type { CalEventResponses } from "@calcom/types/Calendar";
 
 export type VariablesType = {
   eventName?: string;
   organizerName?: string;
   attendeeName?: string;
   attendeeEmail?: string;
   eventDate?: Dayjs;
-  eventTime?: Dayjs;
+  eventEndTime?: Dayjs;
   timeZone?: string;
   location?: string | null;
   additionalNotes?: string | null;
-  responses?: Prisma.JsonValue;
+  responses?: CalEventResponses | null;
   meetingUrl?: string;
+  cancelLink?: string;
+  rescheduleLink?: string;
 };
 
-const customTemplate = async (text: string, variables: VariablesType, locale: string) => {
+const customTemplate = (
+  text: string,
+  variables: VariablesType,
+  locale: string,
+  isBrandingDisabled?: boolean
+) => {
   const translatedDate = new Intl.DateTimeFormat(locale, {
     weekday: "long",
     month: "long",
     day: "numeric",
     year: "numeric",
   }).format(variables.eventDate?.toDate());
 
-  const timeWithTimeZone = `${variables.eventTime?.format("HH:mm")} (${variables.timeZone})`;
   let locationString = variables.location || "";
 
   if (text.includes("{LOCATION}")) {
     locationString = guessEventLocationType(locationString)?.label || locationString;
   }
 
+  const cancelLink = variables.cancelLink ? `${WEBAPP_URL}${variables.cancelLink}` : "";
+  const rescheduleLink = variables.rescheduleLink ? `${WEBAPP_URL}${variables.rescheduleLink}` : "";
+
   let dynamicText = text
     .replaceAll("{EVENT_NAME}", variables.eventName || "")
     .replaceAll("{ORGANIZER}", variables.organizerName || "")
     .replaceAll("{ATTENDEE}", variables.attendeeName || "")
     .replaceAll("{ORGANIZER_NAME}", variables.organizerName || "") //old variable names
     .replaceAll("{ATTENDEE_NAME}", variables.attendeeName || "") //old variable names
     .replaceAll("{EVENT_DATE}", translatedDate)
-    .replaceAll("{EVENT_TIME}", timeWithTimeZone)
+    .replaceAll("{EVENT_TIME}", variables.eventDate?.format("H:mmA") || "")
+    .replaceAll("{EVENT_END_TIME}", variables.eventEndTime?.format("H:mmA") || "")
+    .replaceAll("{TIMEZONE_TIME}", variables.timeZone || "")
```

Comment:
TIMEZONE_TIME and TIMEZONE have same values?

Reply:
<img width="553" alt="Screenshot 2023-04-13 at 7 58 35 PM" src="https://user-images.githubusercontent.com/1780212/231791096-6188aae0-8bef-4482-b231-b1b56457c660.png">
This is confusing. 
- If an attendee receives the email it would be the timezone of attendee ?
- If an organizer receives the email it would be the timezone of organizer ?

Reply:
> This is confusing.
> If an attendee receives the email it would be the timezone of attendee ?
> If an organizer receives the email it would be the timezone of organizer ?

Would "The timezone of the person receiving" be better? Not quite sure how to keep it short but understandable, but I agree that this is a little confusing

Reply:
Yeah I was thinking that only. That makes much more sense